features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/Resources.jsx

```diff
@@ -435,6 +436,13 @@ export default function Resources(props) {
                                     alignItems='stretch'
                                 >
                                     {Object.entries(verbObject).map(([verb, operation]) => {
+                                        let resourcePathName = target.slice(1);
+                                        if (regexPattern.test(resourcePathName)) {
+                                            [resourcePathName] = resourcePathName.split('{');
+                                            if (resourcePathName.endsWith('/')) {
+                                                resourcePathName = resourcePathName.slice(0, -1);
+                                            }
+                                        }
                                         return (
```

Comment:
Ideally, we shouldn't need to update the UI. Let's try to fix the response from the REST API to alter the resource path. Changing the resource name like this in the UI is hacky.

Reply:
As suggested, changed the response from back-end, so that it can be rendered in UI without any changes.. :)